examples/job/get-file-from-job.py

```diff
@@ -94,9 +105,11 @@
 
 # Get some account information
 project_endpoints = ProjectEndpoints(*ENDPOINT_ARGS)
-project_metadata = project_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0]
-project_id = project_metadata['_id']
-owner_id = project_metadata['owner']['_id']
+project_metadata = project_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[
+    0
```

Comment:
This doesn't make sense - shouldn't have a newline here - let's adjust the config

Reply:
Ah, I see. After some searching, I realized that this has to do with the autoformatter's default line length of 88. Here, the character '0' would be the 89th character